Libraries/zephyr/MAX/Include/wrap_max32_sys.h

```diff
@@ -57,7 +57,8 @@ static inline void Wrap_MXC_SYS_SetClockDiv(int div)
 #elif defined(CONFIG_SOC_MAX32690) || defined(CONFIG_SOC_MAX32655) || \
     defined(CONFIG_SOC_MAX32670) || defined(CONFIG_SOC_MAX32672) ||   \
     defined(CONFIG_SOC_MAX32662) || defined(CONFIG_SOC_MAX32675) ||   \
-    defined(CONFIG_SOC_MAX32680) || defined(CONFIG_SOC_MAX32657) || defined(CONFIG_SOC_MAX78002)
+    defined(CONFIG_SOC_MAX32680) || defined(CONFIG_SOC_MAX32657) ||   \
+    defined(CONFIG_SOC_MAX78002) || defined(CONFIG_SOC_MAX78000)
```

Comment:
Please handle clock differences here. "MXC_SYS_CLOCK_ERFO" macro is not defined for MAX78000.